src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const higherDiscount = 50;
+  const lowerDiscount = 20;
+
+  if (days < 3) {
+    return price * days;
+  }
+
+  if (days < 7) {
+    return (price * days) - lowerDiscount;
+  }
+
+  return (price * days) - higherDiscount;
```

Comment:
You calculate `price * days` three times.
Why not to do it once and use variable?

Reply:
Oh yes changes made.